enterprise/admin/views.py

```diff
@@ -62,7 +62,7 @@ def _build_context(self, request, customer_uuid):
         """
         # TODO: pylint acts stupid - find a way around it without suppressing
         enterprise_customer = EnterpriseCustomer.objects.get(uuid=customer_uuid)  # pylint: disable=no-member
-        linked_learners = EnterpriseCustomerUser.objects.filter(enterprise_customer__uuid=enterprise_customer.uuid)
+        linked_learners = EnterpriseCustomerUser.objects.filter(enterprise_customer=enterprise_customer)
```

Comment:
External behavior should be identical. Should being the key word.

It's just a matter of "let's trust the Django ORM to do its job".